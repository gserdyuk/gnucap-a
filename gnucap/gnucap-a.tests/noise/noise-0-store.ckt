circuit to catch memory error in stored command
* during ".store noise" implementation was detected 
v1   1  0  dc 1 ac 1
R1   1  0  100.
R2   1  2  10
R3   2  0  100

.print ac vm(1) vp(1)
.store ac vm(1)
.ac dec 1 1 10
.measure vmmax=max("vm(1)")

.end
