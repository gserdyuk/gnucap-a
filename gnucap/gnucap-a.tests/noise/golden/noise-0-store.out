Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
ac: already installed, replacing
stashing as ac:0
circuit to catch memory error in stored command 
#Freq       vm(1)      vp(1)     
 1.         1.         0.        
 10.        1.         0.        
vmmax= 0.9999998
